src/Umbraco.Infrastructure/Migrations/Upgrade/V_16_3_0/MigrateMediaTypeLabelProperties.cs

```diff
@@ -0,0 +1,199 @@
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Umbraco.Cms.Core;
+using Umbraco.Cms.Core.Configuration.Models;
+using Umbraco.Cms.Core.Models;
+using Umbraco.Cms.Core.Services;
+using Umbraco.Cms.Core.Services.OperationStatus;
+using Umbraco.Cms.Infrastructure.Persistence.Dtos;
+using Umbraco.Extensions;
+
+namespace Umbraco.Cms.Infrastructure.Migrations.Upgrade.V_16_3_0;
+
+[Obsolete("Remove in Umbraco 18.")]
+public class MigrateMediaTypeLabelProperties : AsyncMigrationBase
+{
+    private readonly IMediaTypeService _mediaTypeService;
+
+    private readonly InstallDefaultDataSettings? _dataTypeSettings;
+    private readonly InstallDefaultDataSettings? _mediaTypeSettings;
+
+    private readonly Guid _labelBytesDataTypeKey = new(Constants.DataTypes.Guids.LabelBytes);
+    private readonly Guid _labelPixelsDataTypeKey = new(Constants.DataTypes.Guids.LabelPixels);
+
+    private readonly string[] _mediaTypeKeys =
+    [
+        "cc07b313-0843-4aa8-bbda-871c8da728c8", // Image
+        "4c52d8ab-54e6-40cd-999c-7a5f24903e4d", // File
+        "f6c515bb-653c-4bdc-821c-987729ebe327", // Video
+        "a5ddeee0-8fd8-4cee-a658-6f1fcdb00de3", // Audio
+        "a43e3414-9599-4230-a7d3-943a21b20122", // Article
+        "c4b1efcf-a9d5-41c4-9621-e9d273b52a9c", // Vector Graphics (SVG)
+    ];
```

Comment:
Would it be an idea to simply iterate all media types instead of just the default ones? You're already applying the new data types conditionally, so it should work fine?

The suggestion is linked to the one below.

```suggestion
```